worker/src/RTC/StunPacket.cpp

```diff
@@ -209,6 +209,22 @@ namespace RTC
 					break;
 				}
 
+				case Attribute::NOMINATION:
+				{
+					// Ensure attribute length is 4 bytes.
+					if (attrLength != 4)
+					{
+						MS_WARN_TAG(ice, "attribute NOMINATION must be 4 bytes length, packet discarded");
+
+						delete packet;
+						return nullptr;
+					}
+
+					packet->SetNomination(Utils::Byte::Get4Bytes(attrValuePos, 0));
```

Comment:
This is wrong. According to the draft the NOMINATION attribute is 24 bits long (so 3 bytes).

Reply:
I am wrong. It looks that is 4bytes long even if the value is just 3 bytes. The draft spec sucks.